File: c++/kalkulator.cpp
/*
 * kalkulator.cpp
 */


#include <iostream>

using namespace std;

void dodaj(){
    int a, b;
    cout <<"Podaj dwie liczby: "<< endl;
    cin >> a >> b;
    cout << "Suma: " << a + b<<endl;
    }
void odejmij(){
    int a, b;
    cout <<"Podaj dwie liczby: "<< endl;
    cin >> a >> b;
    cout << "Różnica: " << a - b<<endl;
    }
void pomnoz(){
    int a, b;
    cout <<"Podaj dwie liczby: "<< endl;
    cin >> a >> b;
    cout << "Iloczyn: " << a * b<<endl;
    }
    void podziel(){
    int a, b;
    cout <<"Podaj dwie liczby: "<< endl;
    cin >> a >> b;
    cout << "Iloraz: " << a / b<<endl;
    }
int main(int argc, char **argv)
{
	char operacja;
    cout << "Podaj działanie jakie chcesz wykonać(+, -, *, /)?" << endl;
    cin >> operacja;
    switch(operacja){
        case '+':
            dodaj();
        break;
        case'-':
            odejmij();
        break;
        case '*':
            pomnoz();
        break;
        case'/';
            podziel();
        break;
        default:
            cout << "Nie rozumiem!" << endl;
        break;
        }
	return 0;
}

File: python/kalkulator.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

def pobierzliczbe():
    liczba1 = int(input("Podaj pierwszą liczbe: "))
    return liczba1
    
def pobierzliczbe2():
    liczba2 = int(input("Podaj drugą liczbe: "))
    return liczba2
    
def dodawanie():
    liczba1 = pobierzliczbe()
    liczba2 = pobierzliczbe2()
    suma = liczba1 + liczba2
    print("Suma: ",suma)
    
def odejmowanie():
    liczba1 = pobierzliczbe()
    liczba2 = pobierzliczbe2()
    roznica = liczba1 - liczba2
    print("Różnica: ",roznica)
    
def mnozenie():
    liczba1 = pobierzliczbe()
    liczba2 = pobierzliczbe2()
    iloczyn = liczba1 * liczba2
    print("Iloczyn: ",iloczyn)
    
def dzielenie():
    liczba1 = pobierzliczbe()
    liczba2 = pobierzliczbe2()
    iloraz = liczba1 / liczba2
    print("Iloraz: ",iloraz)
    
    
def main(args):    
    oper = input("Podaj operacje jaką schcesz wykonać +, -, /, * ")
    if oper == "+":
        dodawanie()
    elif oper == "-":
        odejmowanie()
    elif oper == "/":
        dzielenie()
    elif oper == "*":
        mnozenie()
    elif oper != "+" or oper != "-" or oper != "*" or oper != "/":
        print("Nie rozumiem!")
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
